api.py

```
@@ -120,7 +138,8 @@ class BaseTicket(BaseModel):
 
 
 @app.get("/query-stream")
-def qstream(question: Question = Depends()):
+async def qstream(question: Question = Depends()):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)